gui/resources.xml

```diff
@@ -1104,6 +1141,7 @@ Vocabulary
           <property name="icon-name">mypaint-info-symbolic</property>
           <signal name="activate" handler="show_online_help_cb"/>
         </object>
+        <accelerator key="F1" modifiers="GDK_SHIFT_MASK"/>
```

Comment:
Using F1 for help is part of Gnome's HIG so we should look into moving the Scrap Browser functions.